lib/frontend/pages/welcome_page.dart

```diff
@@ -1,44 +1,67 @@
 import 'package:flutter/material.dart';
+import 'package:project_fomo/frontend/components/GradientButton.dart';
 
 //TODO: Recreate welcome page UI
 class WelcomePage extends StatelessWidget {
+  final TextStyle titleStyle = TextStyle(
```

Comment:
Use const instead of final for compile time assignments like this.  I have a more detailed explanation on Emma's PR so check that out for a better understanding of why.